static/elements/chromedash-textarea.js

```diff
@@ -0,0 +1,60 @@
+
+import SlTextarea from '@shoelace-style/shoelace/dist/components/textarea/textarea.js';
+
+class ChromedashTextarea extends SlTextarea {
+  static get properties() {
+    return {
+      ...super.properties,
+      multiple: {type: Boolean},
+      pattern: {type: String},
+      chromedash_single_pattern: {type: String},
+      chromedash_split_pattern: {type: String},
+    };
+  }
+
+  constructor() {
+    super();
+  }
+
+  updated() {
+    if (!this.input) {
+      return;
+    }
+    const value = this.input.value;
+
+    const customCheckValidity = () => {
```

Comment:
Does this need to be a nested function?  I think you could define it as a regular function and then call it as `this.customCheckValidity(value)`.

Reply:
Yes, that is possible now.  I was previously using a more complex process that turned out to not work anyway.

Reply:
Moved into a method, since it uses 'this'.